boards/nucleo-f070/board.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2016 Inria
```

Comment:
side question: is it `INRIA` or `Inria`? Seems that in the RIOT source we have both at the moment, though `INRIA` is the more used one?!